L1Trigger/Configuration/python/L1Trigger_EventContent_cff.py

```diff
@@ -203,6 +216,8 @@ def _appendPhase2Digis(obj):
         'keep *_l1tMETPFProducer_*_*',
         'keep *_l1tNNTauProducer_*_*',
         'keep *_l1tNNTauProducerPuppi_*_*',
+        'keep *_HPSPFTauProducerPF_*_*',
+        'keep *_HPSPFTauProducerPuppi_*_*',
```

Comment:
Shouldn't this be `l1tHPS..`?

Reply:
@cecilecaillol 
Could you please confirm?